Source/WebCore/platform/mediastream/RealtimeMediaSource.cpp

```diff
@@ -402,7 +402,7 @@ bool RealtimeMediaSource::supportsSizeAndFrameRate(std::optional<IntConstraint>
     }
 
     // Each of the non-null values is supported individually, see if they all can be applied at the same time.
-    if (!supportsSizeAndFrameRate(WTFMove(width), WTFMove(height), WTFMove(frameRate))) {
+    if (!supportsSizeAndFrameRate(width, height, WTFMove(frameRate))) {
         // Let's try without frame rate constraint if not mandatory.
         if (frameRateConstraint && !frameRateConstraint->isMandatory() && supportsSizeAndFrameRate(WTFMove(width), WTFMove(height), { }))
```

Comment:
I should note that `supportsSizeAndFrameRate()` doesn't even take rvalue references for any of its parameters (since it doesn't take ownership), so using `WTFMove()` here is probably not even necessary, especially since the `width` and `height` types are `std::optional<int>`.